packages/db-migrations/databases/fxa/patches/patch-122-123.sql

```diff
@@ -0,0 +1,53 @@
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CALL assertPatchLevel('122');
+
+ALTER TABLE emailBounces
+ADD COLUMN emailTypeId BINARY(32) DEFAULT NULL,
+ALGORITHM = INPLACE, LOCK = NONE;
+
+CREATE PROCEDURE `createEmailBounce_2` (
```

Comment:
Do we need `createEmailBounce_2`? I don't see it used in the code.

Reply:
Whoops, I had missed updating it from `createEmailBounce_1`, it's used [here](https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/lib/email/bounces.js#L159) (and now in the fxa-shared model)